src/main/java/seedu/address/logic/commands/SetRsvpCommand.java

```diff
@@ -16,12 +16,14 @@ public class SetRsvpCommand extends UndoableCommand {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Sets the RSVP status of a guest. "
-            + "Parameters: INDEX (must be a positive integer) "
-            + "STATUS (1 - Coming, 2 - Not Coming, 3 - Pending)\n";
+            + "Parameters: INDEX (must be a positive integer and within number of guests in address book) "
+            + "STATUS (s/1 - Coming, s/2 - Not Coming, s/3 - Pending)\n"
+            + "Ex. setrsvp 1 s/1";
```

Comment:
Should we standardise the example format to be "Example: " + COMMAND_WORD + " 1 s/1 ";

Reply:
ok this can be changed